src/modules/auto-sync-state-from-qs.js

```diff
@@ -73,6 +81,9 @@
 		return {
 			page: {
 				fragmentChanged: onFragmentChanged
+			},
+			articleChanger: {
+				enter: onFragmentChanged
```

Comment:
`onFragmentChanged`
https://shft.cl/pc1iwg7x2

Reply:
I know it does the same thing, but my opinion is that we should put it in it's own method with an appropriate name like `onArticleEnter` or something 😁

Reply:
@fhamon check dd4a1ac